k8s/prod/common/sscs/sscs-bulk-scan.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: sscs-bulk-scan
+  namespace: sscs
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: sscs-bulk-scan
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: sscs-bulk-scan
+    version: 0.0.4
+  values:
+    java:
+      replicas: 2
+      memoryRequests: '512Mi'
+      cpuRequests: '250m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1500m'
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "d6c3dd58-af16-425d-8c4a-f491e895e35d"
+      image: hmctspublic.azurecr.io/sscs/sscs-bulk-scan:latest
+      environment:
```

Comment:
i noticed this in your default config:
https://github.com/hmcts/sscs-bulk-scan/blob/master/charts/sscs-bulk-scan/values.yaml#L14-L17
They are very old tactical values which unless you're relying on them should probably be deleted?

Reply:
How can I tell if we are relying on them? Remove and see what happens?

Reply:
probably, it's unlikely you are